docs/blog/2024-09-12-next-gen-module-federation-deployments.md

```diff
@@ -0,0 +1,549 @@
+
+If you don't already have the [GitHub CLI](https://cli.github.com/) installed, I would highly recommend it.
+
+You can then run the following to create your GitHub Repository from your terminal:
+
+```{% commmand="gh repo create" path="~/myorg" %}
+? What would you like to do? Push an existing local repository to GitHub
+? Path to local repository .
+? Repository name nx-zephyr-cloud-example
+? Repository owner Coly010
+? Description Example of Nx React Rspack Module Federation with Zephyr Cloud
+? Visibility Private
+✓ Created repository Coly010/nx-zephyr-cloud-example on GitHub
+  https://github.com/Coly010/nx-zephyr-cloud-example
+? Add a remote? Yes
+? What should the new remote be called? origin
+✓ Added remote https://github.com/Coly010/nx-zephyr-cloud-example.git
+? Would you like to push commits from the current branch to "origin"? Yes
+```
+
+### Step 3: Add Zephyr Cloud
+
+To add Zephyr Cloud itself, we need to install their `zephyr-webpack-plugin`.
+
+{% callout type="note" title="Rspack and Webpack Interop" %}
+The plugin is indeed written for Webpack, however, one of the big advantages of Rspack is the interoperability it offers
+for the Webpack Ecosystem.
+You can learn more about this compatability [here](https://rspack.dev/guide/compatibility/plugin).
+{% /callout %}
+
+Run the following to install the plugin:
+
+```shell
+npm install zephyr-webpack-plugin
+```
+
+Now that the plugin has been installed in our workspace, we need to update the `rspack.config.ts` files to use it.
+This is a very simple two line change:
+
+```ts
+import { withZephyr } from 'zephyr-webpack-plugin';
+
+export default composePlugins(
+  ...,
+  withZephyr()
+)
+```
+
+You'll need to make these updates to the following files:
+
+- `shell/rspack.config.ts`
+- `shell/rspack.config.prod.ts`
+- `remote1/rspack.config.ts`
+
+For example, your `shell/rspack.config.ts` file should now match the following:
+
+```ts
+import { composePlugins, withNx, withReact } from '@nx/rspack';
+import {
+  withModuleFederation,
+  ModuleFederationConfig,
+} from '@nx/rspack/module-federation';
+import { withZephyr } from 'zephyr-webpack-plugin';
+
+import baseConfig from './module-federation.config';
+
+const config: ModuleFederationConfig = {
+  ...baseConfig,
+};
+
+// Nx plugins for rspack to build config object from Nx options and context.
+/**
+ * DTS Plugin is disabled in Nx Workspaces as Nx already provides Typing support for Module Federation
+ * The DTS Plugin can be enabled by setting dts: true
+ * Learn more about the DTS Plugin here: https://module-federation.io/configure/dts.html
+ */
+export default composePlugins(
+  withNx(),
+  withReact(),
+  withModuleFederation(config, { dts: false }),
+  withZephyr()
+);
+```
+
+### Step 4: First Deployment
+
+With all the `rspack.config.ts` files updated, getting your first deployment is as simple as running:
+
+```{% command="npx nx run-many -t build --verbose" path="~/myorg" %}
+> nx run remote1:build:production
```

Comment:
I'd cut out all the output that is not relevant tbh...maybe show the first couple of lines and then `...`